src/styles.scss

```diff
@@ -10,4 +10,108 @@ body {
   margin: 0;
   padding: 0 5px;
   font-family: Roboto, "Helvetica Neue", sans-serif;
+  background-color: black;
+}
+:root {
+  --dl-color-gray-500: #595959;
```

Comment:
What does `dl` stand for?

Reply:
It was supposed to reference Data Catalogue, but I can change it to `dt` to mean Data Tracker